engine/class_modules/sc_rogue.cpp

```diff
@@ -10072,6 +10268,12 @@ void rogue_t::init_spells()
   spell.subterfuge_buff = ( talent.rogue.subterfuge->ok() || talent.outlaw.underhanded_upper_hand->ok() ) ? find_spell( 115192 ) : spell_data_t::not_found();
   spell.vanish_buff = spell.vanish->ok() ? find_spell( 11327 ) : spell_data_t::not_found();
 
+  // Hero Talent Background Spells
+  spell.fatebound_coin_heads_buff = talent.fatebound.hand_of_fate->ok() ? find_spell( 452923 ) : spell_data_t::not_found();
+  spell.fatebound_coin_tails_buff = talent.fatebound.hand_of_fate->ok() ? find_spell( 452917 ) : spell_data_t::not_found();
+  spell.fatebound_coin_tails = talent.fatebound.hand_of_fate->ok() ? find_spell( 452538 ) : spell_data_t::not_found();
```

Comment:
`simc`'s automatic buff tracking let me know this is being affected by `Lethality`'s crit damage bonus right now. No idea why, since it's not a builder. Just something to be aware of, I guess.